### ISSUE TITLE
"vue/require-component-is" false positive

### ISSUE THREAD

**Issue by hjort-e** [nuxt/eslint-config#40]
I'm getting a false positive for the "vue/require-component-is" rule.
Component:
```
<template>
  <Component
    :is="component"
    v-if="component"
    v-bind="{ ...$attrs, $scopedSlots }"
  />
</template>
```
Error:
```
2:3  error  Expected '<component>' elements to have 'v-bind:is' attribute  vue/require-component-is
```
My editors linter handles the rule correctly(no error), but when running the code, it complains no matter what i try.

Someone else had this problem: https://github.com/vuejs/eslint-plugin-vue/issues/869

**Comment by manniL** [comment]
Hey :wave: 
Thanks for reporting. As you already linked, this is related to eslint-plugin-vue which the nuxt eslint config is partially based on. Nothing we can do here, so please raise your voice in https://github.com/vuejs/eslint-plugin-vue/issues/869 :relaxed:

**Comment by ansidev** [comment]
@manniL: Please fix this issue:
Ref: https://github.com/vuejs/eslint-plugin-vue/issues/869#issuecomment-553673273,
